test/filters/test_filters.cpp

```diff
@@ -2061,10 +2061,20 @@ TEST (FrustumCulling, Filters)
   fc.setRegionOfInterest (0.44f, 0.30f, 0.16f, 0.38f);
   fc.setCameraPose (cam2robot);
   fc.filter (*output);
-  // Should extract object; number of points based on milk.pcd
+  // Should extract object; number of points based on milk_cartoon.pcd
```

Comment:
`milk.pcd` is the correct name of the file (https://github.com/PointCloudLibrary/pcl/blob/master/test/milk.pcd). I can't find any `milk_cartoon.pcd`?
Also, where can the file `all.pcd`, that you referenced below, can be found?

Reply:
I am referring to https://github.com/PointCloudLibrary/pcl/blob/master/test/milk_cartoon_all_small_clorox.pcd, which is composed of milk_cartoon, all, and small_clorox.

Reply:
Okay, but if the comment mentions a `milk_cartoon.pcd` or `all.pcd`, I would assume that there are files with these names. Maybe it is clearer to write
`Should extract milk object with 13541 points`
and
`Should extract "all" laundry detergent with 10689 points`